src/jit/legacynonjit/CMakeLists.txt

```diff
@@ -36,7 +36,7 @@ set_property(TARGET legacynonjit APPEND_STRING PROPERTY LINK_DEPENDS ${JIT_EXPOR
 
 set(RYUJIT_LINK_LIBRARIES
    utilcodestaticnohost
-   gcinfo
+   gcinfo_arm
```

Comment:
@BruceForstall this is a bug in the legacynonjit CMakeLists.txt. I've hit it when making my changes that are ARM / ARM64 specific  and was getting missing symbols due to the fact that x86 version of the gcinfo was being linked in.